environments/Cell_profiling_env.yml

```diff
@@ -15,5 +15,5 @@ dependencies:
   - mysqlclient==1.4.6
   - conda-forge::wxpython=4.1.0
   - pip:
-    - cellprofiler==4.2.4
+    - cellprofiler==4.2.6
```

Comment:
ah! Is this as dangerous as I think it could be?

Reply:
Nope!